lib/index.js

```diff
@@ -349,6 +349,20 @@ api.createIndexes = (options, callback) => async.each(
   options, (item, callback) => api.collections[item.collection].createIndex(
     item.fields, item.options, callback), callback);
 
+/**
+ * Creates a streaming GridFS bucket interface.
+ *
+ * By default the writeOptions config value is used for the GridFSBucket
+ * writeConcern option.
+ *
+ * @param options see GridFSBucket documentation
+ * @param callback(err, bucket) called once the operation completes.
+ */
+api.createGridFSBucket = (options, callback) => {
```

Comment:
Why is this async?

Reply:
Hmm.  I did have a weak reason for this, but it doesn't need to be.  I'll change it.